routing/record/validation.go

```diff
@@ -100,3 +112,18 @@ func CheckRecordSig(r *pb.Record, pk ci.PubKey) error {
 	}
 	return nil
 }
+
+type RecordSorter []*pb.Record
+
+func (rs RecordSorter) Less(i, j int) bool {
+	mv, _ := MoreValid(rs[j], rs[i])
```

Comment:
On Sun, Apr 19, 2015 at 11:21:55AM -0700, Jeromy Johnson wrote:
> No, non namespaced keys are not important.  anyone publishing non
> namespaced keys should not expect the correct values or even for the
> value to be stored

So “verify by checking that the payload is signed by the node you're
asking about” still sounds like a valid approach to me (with
subsequent timestamp, serial number, or whatever checks to pick the
most recent signed data).  We can just leave the handling of
non-namespaced keys undefined, and recommend that folks not push or
accept them.